assume/common/units_operator.py

```diff
@@ -214,7 +214,11 @@ async def submit_bids(self, opening: OpeningMessage):
         products = opening["products"]
         market = self.registered_markets[opening["market_id"]]
         logger.debug(f"{self.id} setting bids for {market.name} - {products}")
-        orderbook = await self.formulate_bids(market, products)
+        orderbook = await self.formulate_bids(
+            market=market,
+            products=products,
+            data_dict=self.context.data_dict,
```

Comment:
this is not needed here

Reply:
Why?

Reply:
You never need to pass `self.x` attributes to functions with the same `self`, as you just passed `self.context.data_dict` below instead ;)
So this is addressed